sfm_pc/management/commands/import_google_doc.py

```diff
@@ -746,8 +782,12 @@ def make_relation(self,
         source_position = positions['source']
 
         try:
-            value = data[value_position]
-            value = value.strip()
+            if not date:
+                value = data[value_position]
+                value = value.strip()
+            else:
+                date_parts = [data[value_position + 3], data[value_position + 1], data[value_position + 2]]
+                value = '-'.join(filter(None, date_parts))
```

Comment:
This is a very nice solution. I take it these positions are consistent?

Reply:
they are!